### PR TITLE
feat: contract test coverage for private attributes, context build an…

### PR DESCRIPTION
**Pull request by tanderson-ld**
Added more contract test coverage.

### CODE REVIEW

File: apps/flutter_client_contract_test_service/README.md [hunk 1]
```diff
@@ -20,12 +20,4 @@ If you need to add or modify the OpenAPI yaml files to add or modify functionali
...
-You will also need to patch service_api.openapi.dart to deal with a `Object` vs `dynamic` typing bug.
-
-```
-replace...
-final Map<String, Object> _additionalProperties = <String, Object>{};
-
-with...
-final Map<String, dynamic> _additionalProperties = <String, dynamic>{};
-```
+You will also need to patch service_api.openapi.dart by replacing all instances of `Object` vs `dynamic`.  This is due to a bug in the generator implementation.
```
Comment (kinyoklion) [inline apps/flutter_client_contract_test_service/README.md:23]: Is there an issue for this in the generator repo? We should probably file one if not. Or maybe we will get an alternative.

File: apps/flutter_client_contract_test_service/pubspec.lock [hunk 2]
```diff
@@ -435,18 +435,18 @@ packages:
     dependency: transitive
     description:
       name: material_color_utilities
-      sha256: "0e0a020085b65b6083975e499759762399b4475f766c21668c4ecca34ea74e5a"
+      sha256: "9528f2f296073ff54cb9fee677df673ace1218163c3bc7628093e7eed5203d41"
       url: "https://pub.dev"
     source: hosted
-    version: "0.8.0"
+    version: "0.5.0"
```
Comment (kinyoklion) [inline apps/flutter_client_contract_test_service/pubspec.lock:441]: Strange to see them go down.

File: apps/flutter_client_contract_test_service/testharness-suppressions.txt [hunk 3]
```diff
@@ -1,22 +1,11 @@
-events/context properties/single-kind minimal
-events/context properties/multi-kind minimal
-events/context properties/single-kind with attributes, nothing private
-events/context properties/single-kind, allAttributesPrivate
-events/context properties/single-kind, specific private attributes
-events/context properties/single-kind, private attribute nested property
 events/disabling/evaluation
```
Comment (kinyoklion) [inline apps/flutter_client_contract_test_service/testharness-suppressions.txt:1]: Does this pass after the dynamic change. It seems like it should.

**Review by kinyoklion — APPROVED** [review]

File: apps/flutter_client_contract_test_service/testharness-suppressions.txt (hunk 3 above)
Comment (kinyoklion) [reply apps/flutter_client_contract_test_service/testharness-suppressions.txt:1]: I gave it a try and it did.

Comment (tanderson-ld) [reply apps/flutter_client_contract_test_service/testharness-suppressions.txt]: Removed from suppression.  Thanks.

File: apps/flutter_client_contract_test_service/pubspec.lock (hunk 2 above)
Comment (tanderson-ld) [reply apps/flutter_client_contract_test_service/pubspec.lock]: Reverted.  I'm unsure why this happens on my machine.

File: apps/flutter_client_contract_test_service/README.md (hunk 1 above)
Comment (kinyoklion) [reply apps/flutter_client_contract_test_service/README.md:23]: :+1:

**Pull request merged**